### DISCUSSION TITLE
Modify RED Raw settings for multiple clips hangs- Is there a better way?

### DISCUSSION THREAD

**Discussion by tangierc** [CommandPost/FCPCafe#178] [Final Cut Pro Questions]
Folks, whenever I attempt to modify RED RAW settings for multiple clips in my timeline my computer just hangs to the point of having to force quit FCP (10.6.5).  Does anyone have a more efficient way to do this? I keep reducing the number of clips I am selecting to help with potential memory issues.

M2 Max MacBook Pro / 64 GB Ram / macOS Ventura 13.4

**Comment by latenitefilms** [comment]
You can just use REDCINE-X and update the sidecar files?

**Reply by tangierc** [reply]
Researching. Thanks Chris. I have Redcine-X Pro already so I'll take a look at how to do this. In FCP's inspector for RED RAW settings, I know any changes deletes the proxies. Hopefully modifying the sidecar file won't throw FCP into a fit when launching the project library.

**Reply by latenitefilms** [reply]
FWIW - Modifying the sidecar files will also invalidate the proxies.

**Comment by joema4** [comment]
I just tried it with 15 clips in a simple 23-minute 4096x3112 23.98 fps timeline, a mix of 8k/23.98 and 6k/23.98 from a V-Raptor and Komodo, and it seemed to work OK on my M1 Ultra running FCP 10.6.6 and Ventura 13.4.1. That is with no proxies, and using the latest Red Apple Workflow, ver. 20.0.0, 9-6-22.  Several times I changed all 15 at the same time using FCP "Modify RED RAW Settings". 

There may be some other combination of factors causing it, or other conditions required.

**Comment by tangierc** [comment]
I ended up just doing it in RedCinePro X shortly after originally posting this discussion and talking to Chris. There was some serious latency between me selecting multiple clips in FCP and it applying the changes and maintaining those changes to all of them.
